### ISSUE TITLE
Color contrast issue on UWP vertical scroll bar

### ISSUE THREAD

**Issue by adarshrema** [microsoft/microsoft-ui-xaml#3184]
While investigating the issue below, the root cause seems to be the default XAML scroll bar failing color contrast requirement:
[Color contrast for vertical scrollbar is not meeting the text and graphical component and UI object ratio](https://github.com/microsoft/accessibility-insights-windows/issues/898) reported by @kianrp

We are able to reproduce this in the XAML Controls Gallery app from the Microsoft store. When users are in Light Mode on Windows 10, the default color of the scroll bar (before focus is applied when a user clicks it) is failing the color contrast checker within Accessibility Insights for Windows The ratio is 1.94:1 (the requirement is 3:1).

Screenshot and more information available in the issue linked above.

**Comment by robloo** [comment]
This might be a special case. The slider is also designed to fade out of view and becomes harder to see on purpose.

The contrast analyzer is also checking the foreground/background color as if it was text. No text will be applied in this case. It does fall into the graphical object and UI components section though which is also indicating failure.

![90723284-187fd480-e2da-11ea-849d-b1e0d31c5a95](https://user-images.githubusercontent.com/17993847/90994698-94eb0f80-e587-11ea-934a-49282514141c.png)

**Comment by chingucoding** [comment]
Both light and dark theme scrollbars do not meet the required 3:1 contrast ratio. @YuliKl Can we just use darker/lighter colors to meet the necessary contrast requirements?

**Comment by YuliKl** [comment]
> Both light and dark theme scrollbars do not meet the required 3:1 contrast ratio. @YuliKl Can we just use darker/lighter colors to meet the necessary contrast requirements?

@chingucoding - in general, yes. However, please hold off on making any changes while I work with our design team to help identify better color values.

**Comment by chingucoding** [comment]
Yes of course! If there are changes that can be made, I would gladly make them! Just let me know once you and design know what the best way to fix this would be!

**Comment by YuliKl** [comment]
This issue is resolved with new color/style changes to ScrollBar that are checked into master.
